src/operator/subgraph/mkldnn/mkldnn_subgraph_property.cc

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#if MXNET_USE_MKLDNN == 1
+
+#include "mkldnn_conv_property.h"
+#include "mkldnn_post_quantize_conv_property.h"
+
+namespace mxnet {
+namespace op {
+
+MXNET_REGISTER_SUBGRAPH_PROPERTY(MKLDNN, SgMKLDNNConvProperty);
+MXNET_REGISTER_SUBGRAPH_PROPERTY(MKLDNN_POST_QUANTIZE, SgMKLDNNConvPostQuantizeProperty);
```

Comment:
Can you add a test for this PR. If I understand correctly, the intention is to register multiple properties, under simple backend name. Why are we using MKLDNN and MKLDNN_POST_QUANTIZE here then ? Shouldn't the name be MKLDNN for both ?

Reply:
Yes, this PR is to support register multiple properties under same backend name. They could be all registered under `MKLDNN`. But we decided to separate them because, `MKLDNN` is to cover all subgraph pass that will be used in symbol bind stage, that is, it is designed to optimize common model into MKLDNN specific model. the optimization is suitable for most of networks, and user can use it easily with `export MXNET_SUBGRAPH_BACKEND=MKLDNN `.
`MKLDNN_POST_QUANTIZE` is for optimizing quantized model. it should be used in quantization flow only, by inserting `qsym = qsym.get_backend_symbol('MKLDNN_POST_QUANTIZE')` explicitly.